packages/components/src/checkbox/docs/checkboxfield/disabled.tsx

```diff
@@ -0,0 +1,11 @@
+import { Checkbox, CheckboxField, Text } from "@hopper-ui/components";
+
+export default function Example() {
+    return (
+        <CheckboxField isDisabled>
+            <Checkbox>
+                <Text>Option 1</Text>
```

Comment:
add description here too

Reply:
Why do you think we should add a description? I did not to specifically focus on the size and not add noide.